target_postgres/sinks.py

```diff
@@ -252,3 +255,62 @@ def schema_name(self) -> Optional[str]:
 
         # Schema name not detected.
         return None
+
+    def activate_version(self, new_version: int) -> None:
```

Comment:
I think we could take this function and just drop it into the SDK as it is here, this just has a few tweaks that are needed to make this work in a more generic way